core/src/player/state/states/incapacitaded.rs

```diff
@@ -0,0 +1,79 @@
+use super::*;
+use crate::player::slippery_seaweed::SlipperySeaweed;
+
+pub const ID: Key = key!("core::incapacitaded");
+const SLOWING_SPEED: f32 = 0.3;
+
+pub fn player_state_transition(
+    entities: Res<Entities>,
+    slippery_seaweeds: CompMut<SlipperySeaweed>,
+    collision_world: CollisionWorld,
+    mut player_states: CompMut<PlayerState>,
+) {
+    for (seaweed_ent, _) in entities.iter_with(&slippery_seaweeds) {
```

Comment:
Could you move this code to the `slippery_seaweed.rs` module?

That way if we add new items or things that can send the player into the incapacitated state, they can each be in their own module, and we don't have to keep making this system bigger to account for each new item.